.rubocop.yml

```diff
@@ -389,7 +389,6 @@ Layout/EmptyLineAfterGuardClause:
 # # EOF
 Layout/TrailingEmptyLines:
   Enabled: true
-  EnforcedStyle: final_blank_line
```

Comment:
Removing this is fine.

But I would like to confirm this. Do we need a blank line at end of files or do we end with code/characters or more specifically a `final_newline`?

Note that `newline` and `blank line` are two different entities. Refer to the examples in [this doc](https://www.rubydoc.info/github/bbatsov/RuboCop/RuboCop/Cop/Layout/TrailingEmptyLines).

Reply:
```yml
Layout/TrailingEmptyLines:
  Enabled: true
```
When we keeping only this will require one empty line after the code ends. If we add `final_blank_line`, it'll need one more blank line at the end. 

For reference, you can see the changes done in the `.rb` files.